docs/getting-started.md

```diff
@@ -4,14 +4,18 @@ weight = 3
 nav = [
      "Starting Pilosa",
      "Sample Project",
+     "Using Curl",
+     "Using Go",
+     "Using Java",
+     "Using Python",
      "What's Next?",
 ]
 +++
 
 ## Getting Started
 
 Pilosa supports an HTTP interface which uses JSON by default.
-Any HTTP tool can be used to interact with the Pilosa server. The examples in this documentation will use [curl](https://curl.haxx.se/) which is available by default on many UNIX-like systems including Linux and MacOS. Windows users can download curl [here](https://curl.haxx.se/download.html).
+Any HTTP tool can be used to interact with the Pilosa server. The examples in this documentation will use curl which is available by default on many UNIX-like systems including Linux and MacOS. However, the best way to interface with the Pilosa server is through one of our three official client libraries. Pilosa currently supports [Go](https://github.com/pilosa/go-pilosa), [Java](https://github.com/pilosa/java-pilosa), and [Python](https://github.com/pilosa/python-pilosa).
```

Comment:
I just realized this paragraph, and the one at line 47, are kind of redundant. Is there a reason for repeating this information?

Reply:
This is a paragraph from the original Getting Started file that I edited to fit what we were doing. I don't think we actually need it. I can remove it and then take the Open File Limit comment out of the flag.

Reply:
We should remove one of them, but it seems like a good summary intro for the whole page, so I kind of like it at the beginning.

I don't think the note about the open file limit needs to be changed.